### PR TITLE
Show busy view in Manual Pair view and Password Required modal

### PR DESCRIPTION
**Pull request by woudini**

### CODE REVIEW

File: Blockchain/BackupVerifyViewController.swift [hunk 1]
```diff
@@ -136,7 +136,7 @@ class BackupVerifyViewController: UIViewController, UITextFieldDelegate, SecondP
             }
             if valid {
                 let backupNavigation = self.navigationController as? BackupNavigationViewController
-                backupNavigation?.busyView?.fadeIn()
+                LoadingViewPresenter.sharedInstance().showBusyView(withLoadingText: LocalizationConstants.syncingWallet)
```
Comment (bchrisarriola) [inline Blockchain/BackupVerifyViewController.swift]: Use `LoadingViewPresenter.shared` here

File: Blockchain/Authentication/Views/PasswordRequiredView.swift [hunk 2]
```diff
@@ -76,7 +76,13 @@ class PasswordRequiredView: UIView {
         textFieldPassword.resignFirstResponder()
         textFieldPassword.text = nil
 
-        delegate?.didContinue(with: cleanedPassword)
+        LoadingViewPresenter.shared.showBusyView(withLoadingText: LocalizationConstants.Authentication.downloadingWallet)
+
+        perform(#selector(login), with: cleanedPassword, afterDelay: Constants.Animation.durationLong)
```
Comment (bchrisarriola) [inline Blockchain/Authentication/Views/PasswordRequiredView.swift]: What is this delay for?

File: Blockchain/BCManualPairView.m [hunk 3]
```diff
@@ -135,6 +135,15 @@ - (IBAction)continueClicked:(id)sender
     BlockchainSettings.sharedAppInstance.hasSeenAllCards = YES;
     BlockchainSettings.sharedAppInstance.shouldHideAllCards = YES;
 
+    [[LoadingViewPresenter sharedInstance] showBusyViewWithLoadingText:[LocalizationConstantsObjcBridge downloadingWallet]];
+    
+    [self performSelector:@selector(pair) withObject:nil afterDelay:[ConstantsObjcBridge animationDurationLong]];
```
Comment (bchrisarriola) [inline Blockchain/BCManualPairView.m]: I don't think we should be adding arbitrary delays

File: Blockchain/js/wallet-ios.js [hunk 4]
```diff
@@ -749,8 +749,6 @@ MyWalletPhone.login = function(user_guid, shared_key, resend_code, inputedPasswo
         objc_show_email_authorization_alert();
     }
 
-    objc_loading_start_download_wallet();
```
Comment (bchrisarriola) [inline Blockchain/js/wallet-ios.js]: Remove `loading_start_download_wallet` in obj-c as well?

File: Blockchain/Authentication/Views/PasswordRequiredView.swift (hunk 2 above)
Comment (bchrisarriola) [reply Blockchain/Authentication/Views/PasswordRequiredView.swift]: Doesn't the JSContext always execute in its own thread? I don't think adding an arbitrary delay is an acceptable fix

Comment (bchrisarriola) [reply Blockchain/Authentication/Views/PasswordRequiredView.swift]: I think this bug is probably related to cac69494

Comment (woudini) [reply Blockchain/Authentication/Views/PasswordRequiredView.swift]: I'm open to other solutions but at this point the busy views are not appearing on the manual pair view and password required view when they should be, and the kind of solution that we know will work is concurrency-related.

**Pull request closed without merging**